proto/forch_configuration.proto

```diff
@@ -0,0 +1,25 @@
+/*
+ * Encapsulating Forch configuration
+ */
+syntax = "proto3";
+
+// TODO fill up sections one by one
+message ForchConfig {
+  ProcessConfig process = 1;
+}
+
+message ProcessConfig {
+  int32 scan_interval_sec = 1;
```

Comment:
we should document these since it's an "external" interface into forch, so might as well start adding the comments now.

Reply:
Added